webhook_server/libs/handlers/pull_request_handler.py

```
@@ -880,6 +902,23 @@ async def label_pull_request_by_merge_state(self, pull_request: PullRequest) ->
+        pr_number = await asyncio.to_thread(lambda: pull_request.number)
+
+        self.logger.step(  # type: ignore[attr-defined]
+            f"{self.log_prefix} {format_task_fields('retrigger_checks', 'push_processing', 'processing')} "
+            f"Re-triggering checks for out-of-date PR #{pr_number}",
+        )
+
+        try:
+            # Run configured checks using the shared runner handler method
+            await self.runner_handler.run_retests_from_config(pull_request=pull_request)
+        except Exception:
+            self.logger.exception(f"{self.log_prefix} Failed to re-trigger checks for PR #{pr_number}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)